python/ray/_private/worker.py

```diff
@@ -2449,9 +2475,16 @@ def connect(
         )
         worker.listener_thread.daemon = True
         worker.listener_thread.start()
+        # If the job's logging config is set, don't add the prefix
+        # (task/actor's name and its PID) to the logs.
+        ignore_prefix = global_worker.get_job_logging_config is not None
+        print_to_stdstream_with_ignore = functools.partial(
+            print_to_stdstream, ignore_prefix=ignore_prefix
+        )
+
         if log_to_driver:
             global_worker_stdstream_dispatcher.add_handler(
-                "ray_print_logs", print_to_stdstream
+                "ray_print_logs", print_to_stdstream_with_ignore
```

Comment:
```suggestion
                "ray_print_logs", functools.partial(
            print_to_stdstream, ignore_prefix=ignore_prefix
        )
```

nit, `print_to_stdstream_with_ignore` is not clear, maybe we should just eliminate the naming problem in the first place :)

Reply:
updated: https://github.com/ray-project/ray/pull/47703/commits/0556458613658c85a7619ff90e9a8f450c143ebc